packages/wp-source/src/state.ts

```diff
@@ -1,9 +1,15 @@
 import WpSource from "..";
-import { parsePath } from "./utils/parse-path";
+import { normalize, paramsToRoute } from "./libraries/routeUtils";
 
 const state: WpSource["state"]["source"] = {
-  get: ({ state }) => pathOrLink =>
-    state.source.data[parsePath(pathOrLink).path] || {},
+  get: ({ state }) => routeOrObj => {
```

Comment:
I really like to have a `normalize` util but here I'd try to get rid of the ifs delegating that to the utils, doing something like:
```js
const route = getRoute(routeOrParams);
return state.source.data[route] || {};
```